### PR TITLE
chore(team field): Fix code fences and headings

### PR DESCRIPTION
**Pull request by nschonni**
The code fences had blockquotes inserted before them and didn't render.
Converted them back and pinned them to the left gutter.
The final Credit header had a stuck heading which doesn't render on GH anymore

### CODE REVIEW

File: docs/work/customize/use-team-fields-instead-area-paths.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ ms.date: 04/14/2017
 
 # Use team fields instead of area paths to support teams
 
-[!IN
-CLUDE [temp](../_shared/version-header-tfs-only.md)]
+[!INCLUDE [temp](../_shared/version-header-tfs-only.md)]
```
Comment (nschonni) [inline docs/work/customize/use-team-fields-instead-area-paths.md]: Note sure what this was, but looked broken

File: docs/work/customize/use-team-fields-instead-area-paths.md [hunk 2]
```diff
@@ -225,17 +222,19 @@ For backlog items you create from a team's backlog page, TFS assigns the default
 
 2.  [Download the process template](../work-items/guidance/manage-process-templates.md) that corresponds to the template used to create your team project.
 
-	> [!IMPORTANT]  
-	>Make sure that you download the process template from the upgraded server. Also, the Visual Studio client version you use for both the download process and using **witadmin** must match the server version. For example, if you have upgraded to TFS 2015, you need to work from Visual Studio 2015. If you use an older version of Visuals Studio, you may get errors during the upload process. 
+  > [!IMPORTANT]
```
Comment (nschonni) [inline docs/work/customize/use-team-fields-instead-area-paths.md]: This got caught when I did the find/replace for tabs/spaces but can be reverted

**Comment by steved0x** [comment]
@nschonni Thank you for your contribution, I have assigned this to the author of the article for review. Thanks again!

**Comment by KathrynEE** [comment]
@nschonni - Thank you for this contribution. I've confirmed your changes.

**Pull request merged**